frontend/src/components/Question/Question.scss

```diff
@@ -28,7 +28,7 @@
             font-size: 1.3em;
         }
 
-        &:not(.title){
+        &:not(.emphasize-title){
             text-align: center;
         }
     }
```

Comment:
I don't know the entire context, but wouldn't it be better to align to the center by default and do something when `.emphasize-title` is present?